msg/Classification.msg

```diff
@@ -10,6 +10,4 @@ std_msgs/Header header
 #   user-defined threshold.
 ObjectHypothesis[] results
 
-# The 2D data that generated these results (i.e. region proposal cropped out of
-#   the image). Not required for all use cases, so it may be empty.
-sensor_msgs/Image source_img
\ No newline at end of file
+# Source data that generated this classification are not a part of the message. If you need to access them, use an exact or approximate time synchronizer in your code, as this message's header should match the header of the source data.
```

Comment:
Should we move the new comment to the README? Same for Detection2D and Detection3D.

Reply:
I'll add it to README, but leave it here, because this place is IMO much more visible than the README.